crates/daktilo_lib/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "daktilo_lib"
+version.workspace = true
+description.workspace = true
+authors.workspace = true
+license.workspace = true
+readme.workspace = true
+homepage.workspace = true
+repository.workspace = true
+keywords.workspace = true
+categories.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+
+[dependencies]
+rdev = { version = "0.5.3", features = ["serialize"] }
+rodio = { version = "0.17.3", default-features = false, features = [
+  "symphonia-mp3",
+] }
+rust-embed = { version = "8.0.0", features = ["debug-embed", "compression"] }
+toml = "0.8.6"
+fastrand = "2.0.1"
+dirs = "5.0.1"
+regex = "1.10.2"
+serde_regex = "1.1.0"
+tokio.workspace = true
+serde.workspace = true
+thiserror.workspace = true
+tracing.workspace = true
+tracing-subscriber.workspace = true
+# To be moved to cli? Not core lib functionality
+colored.workspace = true
```

Comment:
We should probably make this a workspace dependency since it is used in both crates.

Reply:
It is a workspace (`workspace = true`) dependency. But I think the (pretty) printing should be in the cli's responsibility.